docs/content/docs/build/tokenproviders.md

```diff
@@ -90,50 +90,56 @@ import { generateToken } from "@fluidframework/azure-service-utils";
 //Replace "myTenantKey" with your key here.
 const key = "myTenantKey";
 
-const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
-    // tenantId, documentId, userId and userName are required parameters
-    const tenantId = (req.query.tenantId || (req.body && req.body.tenantId)) as string;
-    const documentId = (req.query.documentId || (req.body && req.body.documentId)) as string;
-    const userId = (req.query.userId || (req.body && req.body.userId)) as string;
-    const userName = (req.query.userName || (req.body && req.body.userName)) as string;
-    const scopes = (req.query.scopes || (req.body && req.body.scopes)) as ScopeType[];
-
-    if (!tenantId) {
-        context.res = {
-            status: 400,
-            body: "No tenantId provided in query params",
-        };
-    }
-
-    if (!key) {
-        context.res = {
-            status: 404,
-            body: `No key found for the provided tenantId: ${tenantId}`,
-        };
-    }
-
-    if (!documentId) {
-        context.res = {
-            status: 400,
-            body: "No documentId provided in query params"
-        };
-    }
-
-    let user = { name: userName, id: userId };
-
-    // Will generate the token and returned by an ITokenProvider implementation to use with the AzureClient.
-    const token = generateToken(
-        tenantId,
-        documentId,
-        key,
-        scopes ?? [ScopeType.DocRead, ScopeType.DocWrite, ScopeType.SummaryWrite],
-        user
-    );
+const httpTrigger: AzureFunction = async function (
```

Comment:
what actually changed here?  i also don't think this section should be changed to 2 space tabs since that's inconsistent with the rest of this document and our codebase in general

Reply:
Restored it back to four space formatting. I still think the formatting pass helps make the code more readable though.

As for what changed:
- the function doesn't throw an error if document ID is provided
-  it now parses for additionalDetails as a parameter in the request. It didn't before and if you used this code snippet before, you would undefined back in your `member.additionalDetails` in your audience

Reply:
Yeah, watch the formatting. Do you have the [editorconfig VSCode extension installed](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)? It _should_ help auto-configure this, but if it's not, I'd like to know. :)

Reply:
I do! And it works just fine when working in our repo 😄 Here though, we are copying this code into the output for the Azure Function starter template so it picks up different rules

Reply:
Ahhh, yes... consistent formatting in the docs is a pain at the moment.